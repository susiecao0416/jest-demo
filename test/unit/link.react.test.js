import React from 'react';
import Link from '../../src/link.react';
import renderer from 'react-test-renderer';

it('renders as an anchor when no page is set', () => {
	const tree = renderer.create(<Link page='www.face.com'>Facebook</Link>).toJSON();
	expect(tree).toMatchSnapshot();
});