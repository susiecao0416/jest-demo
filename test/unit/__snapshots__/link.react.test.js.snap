// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as an anchor when no page is set 1`] = `
<a
  className="normal"
  href="www.face.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
